### PR TITLE
Feat/meta tags

### PR DESCRIPTION
**Pull request by trinegl**
Meta image must be updated before merge! Waiting for correct image.
Also fixed the offering/our offer header (color and wording)

### CODE REVIEW

File: _includes/head.liquid [hunk 1]
```diff
@@ -6,4 +6,26 @@
...
+  <meta name="description" content="Born from digital, evolving over 2 decades, SinnerSchrader is a leading
+  design and engineering agency headquartered in Hamburg, Germany.">
+
+  <!-- Open Graph / Facebook -->
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="https://sinnerschrader.com/">
+  <meta property="og:title" content="SinnerSchrader | Experience Design and Engineering Agency.">
+  <meta property="og:description" content="Born from digital, evolving over 2 decades, SinnerSchrader is a leading
+  design and engineering agency headquartered in Hamburg, Germany.">
+  <meta property="og:image" content="https://sinnerschrader.com/static/sinnerschrader.png">
```
Comment (timbru31) [inline _includes/head.liquid:22]: Our other pictures are in the `/assets/` folder- is this intended?

Comment (trinegl) [reply _includes/head.liquid:22]: Yes, this is the reason for the draft PR status, I'm waiting for the correct image

Comment (timbru31) [reply _includes/head.liquid:22]: Alright, just wanted to ensure the sharing will work correctly.

File: styles/core/_base.scss [hunk 2]
```diff
@@ -33,6 +33,12 @@ h1 {
 	font-weight: 200;
 }
 
+h2 {
+	.is-dark & {
+		color: $color-primary--white;
+	}
+}
+
```
Comment (luismeyer) [inline styles/core/_base.scss:41]: i think this was already fixed right?

Comment (trinegl) [reply styles/core/_base.scss]: Yes, deleted this part

**Review by timbru31 — APPROVED** [review]

**Pull request merged**